src/convertToObject.js

```diff
@@ -7,6 +7,19 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const stylesLines = sourceString.split(';').map((item) => item.trim());
+
+  const cssObject = stylesLines.reduce((acc, line) => {
+    const [keys, values] = line.split(':');
+
+    if (keys && values) {
+      acc[keys.trim()] = values.trim();
+    }
```

Comment:
```suggestion
    if (keys && values) {
    const trimmedKey = key.trim();
    const trimmedValue = value.trim();
    
      acc[trimmedKey] = trimmedValue;
    }
```
use more variables ;)